pkgs/misc/vim-plugins/default.nix

```diff
@@ -1863,7 +1863,7 @@ rec {
       rev = "f0d63579eab7548cf12f979dc52ef5a370ecbe63";
       sha256 = "06nix49j4inxy3rkcv32f4ka89g4crqwfqnrm3b76iwwky8m2p17";
     };
-    dependencies = [];
+    dependencies = ["WebAPI"];
```

Comment:
This will get overwritten unless persisted in `nixpkgs/pkgs/misc/vim-plugins/vim2nix/additional-nix-code/`. Also I am not sure how this would work for the `dependencies` field that is usually interfered by `vim2nix`. If there is no easy way to do this it is better to remove this plugin from `nixpkgs/pkgs/misc/vim-plugins/vim-plugin-names` and move it up in the file so it will not be automatically updated.

Reply:
Dependencies are usually found by a file like this in the plugin repository: https://github.com/MarcWeber/vim-addon-manager/blob/master/addon-info.json 
You could make a pull request for `gist-vim` to add this file. Otherwise you can stick to the alternative I described before.

Reply:
Thank you @Mic92 for the review. I filed a PR upstream, if @mattn is not willing to add it upstream, my other options would be to extend [nix#NixDerivation](https://github.com/NixOS/nixpkgs/blob/ee57cb3a2479d894ca407d32217bd4988482bf66/pkgs/misc/vim-plugins/vim2nix/autoload/nix.vim#L33-L123) to support a concept of `additional_dependencies`.

Reply:
@Mic92 problem with that is it's text substitution and `dependencies = [];` already gets defined 😕

Reply:
Yes I would love to finish my rewrite some time, so we could get rid of those stupid limitations.